app/tasks/maintenance.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+global $cli;
+
+require_once __DIR__.'/../init.php';
+
+use Appwrite\Database\Database;
+use Appwrite\Database\Document;
+use Appwrite\Database\Adapter\MySQL as MySQLAdapter;
+use Appwrite\Database\Adapter\Redis as RedisAdapter;
+use Appwrite\Database\Validator\Authorization;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Config\Config;
+
+
+function getConsoleDB() {
+    global $register;
+    $consoleDB = new Database();
+    $consoleDB->setAdapter(new RedisAdapter(new MySQLAdapter($register), $register));
+    $consoleDB->setNamespace('app_console'); // Main DB
+    $consoleDB->setMocks(Config::getParam('collections', []));
+    return $consoleDB;
+}
+
+$cli
+    ->task('maintenance')
+    ->desc('Schedules maintenance tasks and publishes them to resque')
+    ->action(function () {
+        // Convert string to integer
+        $interval = App::getEnv('_APP_MAINTENANCE_INTERVAL', '') + 0;
+        //Convert Seconds to microseconds
+        $interval = $interval * 1000000;
```

Comment:
Add a comment with the # of days to $interval value

Reply:
Actually interval represents # days in seconds (1 day = 86400s)

Reply:
Yeh, I got it, just would be easier to read the code if you had a comment beside the large number.

Reply:
Sure